sagemaker-pipelines/step-decorator/quick-start/modular/requirements.txt

```diff
@@ -1,5 +1,5 @@
 scikit-learn
-xgboost==1.7.1
+xgboost>=1.7.1,<2
```

Comment:
Thanks for the review!
I tried `xgboost>=1.7.1` initially and the execution failed in the model registry step because the ModelBuilder is unable to auto detect DLC image version as per xgboost >= 2 and the python version 3.10.
Thus I restrict the xgboost version to be <2 to make it pass.